configs/crashtests-webrtc.ini

```diff
@@ -0,0 +1,28 @@
+[runtests]
+# Settings related to executing runtestsremote.py
+
+# test_name is a descriptor used by runtestsremote.py to
+# determine which of the downloaded unit tests to run.
+#
+# test_manifest is the path to the location of the test_manifest.
+#
+# test_package_names is a list of the keys for test_packages.json
+# which will need to be downloaded in order to run the test.
+
+test_name = crashtest
+test_manifest = reftest/tests/testing/crashtest/autophone-webrtc.list
```

Comment:
I know it is redundant with the parent directory but to follow the adjacent file, lets call it autophone-crashtests-webrtc.list